samples/time-spel-log/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '3.2.1'
+    id 'org.springframework.boot' version '3.2.3'
```

Comment:
Yeah... There is no way to feed Dependabot with something like `directory: /samples/*` to make it look into every single sample for updates.
And on the other hand I don't want to rely in samples on the property from the root project.
The point of the sample is to be fully working independent project.

I'm merging this one for now, but I'm opened for any other ideas how to automate this.

Thanks

Reply:
We can instead pass it in when we invoke the sample like we do in Spring Pulsar [samples](https://github.com/spring-projects/spring-pulsar/blob/1f2f6ed46d66a8cb7a146a1e4a742cfead6e0c2b/spring-pulsar-sample-apps/sample-apps-check-ci.gradle#L1-L12).

Reply:
That's totally different task.
The goal of the change your have done here and we are discussing is to be able upgrade these deps in samples smoothly.
We probably can have some generated Dependabot config or write some smart Gradle task.
I wish there was a Dependabot GH Action where we would feed it with config what to check for upgrades...

Reply:
>That's totally different task.
>The goal of the change your have done here and we are discussing is to be able upgrade these deps in samples smoothly

Notice that the sample init script uses the `rootProject.springBootVersion` property as the version of the SB plugin. 
```groovy
        settings.pluginManagement {
                resolutionStrategy {
                        eachPlugin {
                                if (requested.id.id == "org.springframework.boot") {
                                        useVersion("$rootProject.springBootVersion")
                                }
                        }
                }
        }
```
And because that property does get auto-upgraded then the plugin in the sample also does (indirectly). 
Not quite the same as having the samples deps all get auto-upgraded - but better than having it static on some  boot version.

>I wish there was a Dependabot GH Action where we would feed it with config what to check for upgrades...

:nod:

Reply:
Right. But again: the point is to have samples as fully independent project. So, community could use them as is without any actions to replace that placeholder manually.